src/sentry/lang/javascript/errormapping.py

```diff
@@ -112,15 +113,25 @@ def rewrite_exception(data):
     in place and returns `True` if a modification was performed or `False`
     otherwise.
     """
+    meta = Meta(data.get("_meta"))
     rv = False
-    for exc in get_path(data, "exception", "values", filter=True, default=()):
+
+    values_meta = meta.enter("exception", "values")
+    for index, exc in enumerate(get_path(data, "exception", "values", filter=True, default=())):
```

Comment:
`filter=True` means you will skip entries that are `None` (removed as invalid during ingestion). That means your index is out of sync with the index in data. I'd remove the filter and instead skip inside the loop.